roles/etcd/templates/etcd.j2

```diff
@@ -131,10 +131,11 @@ init_update_cluster() {
     # on master nodes, if the cluster is being initialized for first time then initialize it
     peers=""
     get_peers "{{ get_peer_addr() }}"
+    rm -rf ${ETCD_DATA_DIR}/wal/*
```

Comment:
Will this remove the etcd data directory even for the case of cluster getting updated ? That may not be what we want.

Reply:
No, it won't delete data directory on update. it's only cleanup directory whenever ETCD node/container will  start or restart. During debugging it has been found that restart node/container was not able to join cluster.
This work around would let the restart ETCD node to rejoin the cluster again. Looks like this is know issue in ETCD . Found following - 
“etcd process hanging after removed from membership and restart #7512”

Reply:
ok, sounds good. 
LGTM